src/components/navigation/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import { NavPrivate, NavPublic } from ".";
+
+import { isLoggedIn } from "../../services/auth"
+
+export { NavPrivate } from "./private"
+export { NavPublic } from "./public"
+
+export const NavBar = () => {
+  return (
+    <div
+      style={{
```

Comment:
For now style in jsx is ok, but consider to move it to styles file.

Reply:
okay